testing/acceptance_test.py

```diff
@@ -1584,3 +1584,27 @@ def test_no_terminal_plugin(pytester: Pytester) -> None:
     pytester.makepyfile("def test(): assert 1 == 2")
     result = pytester.runpytest("-pno:terminal", "-s")
     assert result.ret == ExitCode.TESTS_FAILED
+
+def test_get_exception_on_teardown_failure(pytester: Pytester) -> None:
+    """Smoke test to be sure teardown exceptions handled properly via node property"""
+    pytester.makepyfile(
+        conftest="""
+        import sys
+        import pytest
+        def pytest_exception_interact(node, call, report):
+            sys.stderr.write("{}".format(node.teardown_exceptions))
+
+        import pytest
+        @pytest.fixture
+        def mylist():
+            yield
+            raise AssertionError(111)
+        """,
+        test_file="""
+        def test_func(mylist):
+            assert True
+        """,
+    )
+    result = pytester.runpytest()
+    assert result.ret == ExitCode.TESTS_FAILED
+    assert "AssertionError(111)" in result.stderr.str()
```

Comment:
There is a `result.stderr.fnmatch_lines()` used to check for expected text (ex. line 89)

Reply:
I see, `.str()` is used too (ex. line 260), however, as `not in` check. Okay, let's leave as it is, will change if needed